Solutions/NozomiNetworks/Data Connectors/Connector_CEF_NozomiNetworks.json

```diff
@@ -0,0 +1,121 @@
+{
+    "id": "NozomiNetworks",
+    "title": "Nozomi Networks",
```

Comment:
Title should be in the format "PROVIDER NAME APPLIANCE NAME". Appliance Name is missing.

Reply:
done